openshift_tools/monitoring/hawk_sender.py

```diff
@@ -0,0 +1,190 @@
+        currently has stored
+        """
+
+        print "\nHawkSender Key/Value pairs:"
+        print "=============================="
+        for unique_metric in self.unique_metrics:
+            print("%s:  %s") % (unique_metric.key, unique_metric.value)
+        print "==============================\n"
+
+    def print_unique_metrics(self):
+        """
+        This function prints all of the information of the UniqueMetrics that HawkSender
+        currently has stored
+        """
+
+        print "\nHawkSender UniqueMetrics:"
+        print "=============================="
+        for unique_metric in self.unique_metrics:
+            print unique_metric
+        print "==============================\n"
+
+    def parse_config(self):
+        """ parse default config file """
+
+        if not self.config:
+            if not os.path.exists(self.config_file):
+                raise HawkSenderException(self.config_file + " does not exist.")
+            self.config = yaml.load(file(self.config_file))
+
+    def get_default_host(self):
+        """ get the 'host' value from the config file """
+        self.parse_config()
+
+        return self.config['host']['name']
+
+    def get_default_hawk_connecton(self):
+        """ get the values and create a hawk_connection """
+
+        self.parse_config()
+        hawk_server = self.config['hawk']['url']
+        hawk_user = self.config['hawk']['user']
+        hawk_password = self.config['hawk']['pass']
+        hawk_ssl_verify = self.config['hawk'].get('ssl_verify', False)
+        hawk_debug = self.config['hawk'].get('debug', False)
+        hawk_active = self.config['hawk'].get('active', True)
+
+        if isinstance(hawk_ssl_verify, str):
+            hawk_ssl_verify = (hawk_ssl_verify == 'True')
+
+        if self.debug:
+            hawk_debug = self.debug
+        elif isinstance(hawk_debug, str):
+            hawk_debug = (hawk_debug == 'True')
+
+        hawk_connection = HawkConnection(url=hawk_server,
+                                         user=hawk_user,
+                                         password=hawk_password,
+                                         ssl_verify=hawk_ssl_verify,
+                                         debug=hawk_debug,
+                                         active=hawk_active,
+                                        )
+
+        return hawk_connection
+
+    def add_heartbeat(self, heartbeat, host=None):
+        """ create a heartbeat unique metric to send to hawk """
+
+        if not host:
+            host = self.host
+
+        hb_metric = UniqueMetric.create_heartbeat(host,
+                                                  heartbeat.templates,
+                                                  heartbeat.hostgroups,
+                                                 )
+        self.unique_metrics.append(hb_metric)
+
+    def add_zabbix_keys(self, zabbix_keys, host=None, synthetic=False):
```

Comment:
We don't want any references to zabbix in hawk stuff.

Rename function `add_zabbix_keys` to something like `add_metric`.

Rename `zabbix_keys` to something like `metrics`